source/fundamentals/builders/aggregates.txt

```diff
@@ -700,15 +635,17 @@ Use the ``bucket()`` method to create a :manual:`$bucket </reference/operator/ag
 pipeline stage that automates the bucketing of data around predefined boundary
 values.
 
-The following example creates a pipeline stage that groups incoming documents based
+The following examples use data modeled with the following Kotlin data class:
+
+.. literalinclude:: /examples/generated/AggregatesBuilderTest.snippet.bucket-data-class.kt
```

Comment:
a few sections use this example even though it's only defined in this section. can you redefine it in other sections where it's used?

Reply:
Added